.claude/scripts/pm/prd-list.sh

```
@@ -0,0 +1,89 @@
+# !/bin/bash
```

Shellcheck (0.10.0)

[error] Remove spaces between # and ! in the shebang.

(SC1115)